packages/core/dist/GondelComponent.d.ts

```diff
@@ -10,6 +17,10 @@ export interface GondelComponent {
     sync?(): void;
 }
 export declare class GondelBaseComponent implements GondelComponent {
+    /**
+     * The components initial identification mappings
```

Comment:
`The components initial identifcation mappings` seems to be a little bit hard to understand..

What about "An internal mapping to lookup the registered name for each namespace"?